go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/cch123/elasticsql
+module github.com/lemonyxk/elasticsql
```

Comment:
package path should not be changed

Reply:
sorry my mistake. i will close this PR and open a new one.